common/scripted_modifiers/wc_portrait_modifiers.txt

```diff
@@ -38,6 +38,14 @@
 		}
 	}
 	
+	modifier = { #Vrykul always wear beards
+		add = -50
+		OR = {
+			has_trait = creature_vrykul
+			culture = { has_graphical_culture = vrykul_portrait_gfx }
+		}
+	}
+	
```

Comment:
I wouldn't say always.
I would say minus from `portrait_northern_clothing_trigger` will be enough, so this part isn't needed.
https://wow.gamepedia.com/Iskalder

Reply:
> 
> 
> I wouldn't say always.
> I would say minus from `portrait_northern_clothing_trigger` will be enough, so this part isn't needed.
> https://wow.gamepedia.com/Iskalder

We added this because we tested it and there were too many beardless Vrykul in-game when using the default minus from `portrait_northern_clothing_trigger`. I don't think it's a good idea.

Reply:
I see.